ansible-requirements.txt

```diff
@@ -1,3 +1,5 @@
 ansible>=7.0.0
 importlib-metadata
 jsonschema  # MIT
+oauthlib>=3.2.0 # k8s lib that requires manual upgrade
+kubernetes
```

Comment:
https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html
The below requirements are needed **on the host that executes** this module.